frontend/src/routes/register/+page.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+    import {Page} from '$lib/layout';
+    import t from '$lib/i18n';
+    import {Input, ProtectedForm, lexSuperForm, lexSuperValidate} from '$lib/forms';
+    import {z} from 'zod';
+    import Button from '$lib/forms/Button.svelte';
+    import {register} from '$lib/user';
+    import {goto} from '$app/navigation';
+
+    const formSchema = z.object({
+        name: z.string().min(1, $t('register.name_missing')),
+        email: z.string().email($t('register.email')),
+        password: z.string().min(1, $t('register.password_missing')),
+    });
+    let {form, errors, valid, update, reset, message} = lexSuperForm(formSchema);
+    let turnstileToken = '';
+
+    async function submit() {
+        await lexSuperValidate($form, formSchema, update);
+        if (!$valid) return;
+        const {user, error} = await register($form.password, $form.name, $form.email, turnstileToken);
+        if (error) {
+			if (error.turnstile) {
```

Comment:
mind indenting these a little more?

Reply:
I can't tell if you're being sarcastic or not, it seems fine to me.

Reply:
sorry it came across that way, mine looks like this:
![image](https://user-images.githubusercontent.com/4412848/234876065-4f874b8a-648b-4f99-9cab-04a0c5ea0ab0.png)

L23 is a one space indent where everything else is 4 I believe.

Reply:
thanks. That's really weird, and now on github it looks like it's indented too much, very strange. It looks like for some reason tabs and spaces got mixed. All the lines of code are using spaces, except L23-L28. I have no idea why. I'll fix it.